egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -16,8 +16,12 @@ decode_dir_affix=
 num_epochs=4
 remove_egs=false
 common_egs_dir=
-
 affix=
+
+# decoding options
+extra_left_context=50
```

Comment:
I assume you mean chunk_left_context-- I'd say better to do
```
extra_left_context=$[$chunk_left_context+10]
```
.. no conditionals as not really needed, and $[] is used in most of the scripts for basic math like this.